frontend/toolkit-webapp/src/pages/ResetPassword.js

```diff
@@ -0,0 +1,162 @@
+        }
+        else
+        {
+            alert('Something went wrong! Try inputting your new password again')
+            return
+        }
+       
+    }
+
+    const returnToLogin = (e) => {
+        props.history.push('../login');
+    }
+
+    return (
+        <div style={{height:'100vh', width:'100vw'}}>
+            <TopNavBar hideComponents={true}/>
+            <div className={classes.darkOverlay}>
+                <video className={classes.video} autoPlay loop muted playsInline>
+                    <source src={videoSource} type="video/mp4" />
+                Your browser does not support the video tag.
+                </video>
+            </div>
+            <div className={classes.container}>
+                <div className={classes.block}>
+                    <form autoComplete="off" onSubmit={onSubmit}>
+                        <Paper className={classes.paper} elevation={5} square={false}>
+                        <Avatar className={classes.avatar}>
+                            <LockOutlinedIcon />
+                        </Avatar>
+                            <Box m={2} pt={2} style={{justifyContent: 'center'}}>
+                                <Typography className={classes.Title} variant="h5">Reset Your Password</Typography>
+                            </Box>
+                            <div className={classes.TextBox}>
+                                <TextField color='primary'
+                                    size='small'
+                                    variant="filled"
+                                    label='New Password'
+                                    type="password"
+                                    value={password}
+                                    onChange={e => setPassword(e.target.value)}
+                                    margin="normal"
+                                    required={true}
+                                    fullWidth
+                                    style={{backgroundColor: "rgba(255,255,255,0.8)"}}
+                                />
+
+                                <TextField
+                                    size='small'
+                                    variant="filled"
+                                    label='Confirm Password'
+                                    type="password"
+                                    value={passwordCopy}
+                                    onChange={e => setPasswordCopy(e.target.value)}
+                                    margin="normal"
+                                    required={true}
+                                    fullWidth
+                                    style={{backgroundColor: "rgba(255,255,255,0.8)"}}
+                                />
+
+                            </div>
+                            <Button fullWidth type='submit' className={classes.resetPassButton} size="medium" variant="contained" >
+                                Reset Password
```

Comment:
Center text inside of div box. Right now it's pushed to the left.

Reply:
I believe I fixed it? Can't really tell if the text isn't in the middle rn